wp-cli-templates/block-index-js.mustache

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ExternalLink, Path } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import renderMaterialIcon from '../../utils/render-material-icon';
+{{#externalEdit}}
+import edit from './edit';
+{{/externalEdit}}
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name     = '{{ slug }}';
+export const title    = __( '{{ title }}', 'jetpack' );
+export const settings = {
+	title,
+
+	description: (
+		<Fragment>
+			<p>{ __( '{{ description }}', 'jetpack' ) }</p>
+			<ExternalLink href="#">{ __( 'Learn more about {{ title }}', 'jetpack' ) }</ExternalLink>
+		</Fragment>
+	),
+
+	/* @TODO add icon */
+	icon: renderMaterialIcon(
+		<Path d="M9 15h2V9H9v6zm1-10c-.5 0-1 .5-1 1s.5 1 1 1 1-.5 1-1-.5-1-1-1zm0-4c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9zm0 16c-3.9 0-7-3.1-7-7s3.1-7 7-7 7 3.1 7 7-3.1 7-7 7z" />
+	),
+
+	category: 'jetpack',
+
+	/* @TODO add keywords */
+	keywords: [ {{#keywords}}__( '{{ keyword }}', 'jetpack' ), {{/keywords}} ],
+
+	supports: {
+		html: false,
+	},
+
+	/* @TODO edit */
+	{{#externalEdit}}edit,{{/externalEdit}}
+	{{^externalEdit}}edit: () => <div>Hi! The edit view for this block goes here.</div>,{{/externalEdit}}
```

Comment:
I would just keep CLI and these files more simple and either default to having external or inline example.

Maybe we can leave edit as an external (they'll always be complicated enough to justify that) and have `save` return `<p>Block output goes here</p>` ?

What do you think?